File: Curso Programação Orientada a Objetos/Java com POO/Aula09(Exercicio Pratico)/src/teste/Pessoa.java
package teste;

public class Pessoa {
    //Atributos
    private String nome;
    private int idade;
    private String sexo;
    
    //Métodos
    public void fazerAniver() {
        this.setIdade(this.getIdade() + 1);
        //this.idade++; -> serve como uma opção também
    }
    
    //Construtor
    public Pessoa(String nome, int idade, String sexo) {
        this.nome = nome;
        this.idade = idade;
        this.sexo = sexo;
    }
    
    //Getters e Setters
    //nome
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }

    //idade
    public int getIdade() {
        return idade;
    }
    public void setIdade(int idade) {
        this.idade = idade;
    }

    //sexo
    public String getSexo() {
        return sexo;
    }
    public void setSexo(String sexo) {
        this.sexo = sexo;
    }
    
}

File: Curso Programação Orientada a Objetos/PHP com POO/Aula09-ExercicioPratico/Pessoa.php
<?php

class Pessoa {
    //Atributos
    private $nome;
    private $idade;
    private $sexo;
    
    //Métodos
    public function fazerAniver() {
        //$this->idade++;
        $this->setIdade($this->getIdade() + 1);       
    }
    
    //Construtor
    public function __construct($nome, $idade, $sexo) {
        $this->nome = $nome;
        $this->idade = $idade;
        $this->sexo = $sexo;
    }
    
    //Getters
    public function getNome() {
        return $this->nome;
    }

    public function getIdade() {
        return $this->idade;
    }

    public function getSexo() {
        return $this->sexo;
    }
    
    
    //Setters
    public function setNome($nome) {
        $this->nome = $nome;
    }

    public function setIdade($idade) {
        $this->idade = $idade;
    }

    public function setSexo($sexo) {
        $this->sexo = $sexo;
    }

}
